lib/common.sh

```diff
@@ -199,8 +199,8 @@ export DEFAULT_HOSTS_MEMORY=${DEFAULT_HOSTS_MEMORY:-4096}
 # Cluster
 export CLUSTER_NAME=${CLUSTER_NAME:-"test1"}
 export CLUSTER_APIENDPOINT_IP=${CLUSTER_APIENDPOINT_IP:-"192.168.111.249"}
-export KUBERNETES_VERSION=${KUBERNETES_VERSION:-"v1.21.2"}
-export UPGRADED_K8S_VERSION=${UPGRADED_K8S_VERSION:-"v1.21.2"}
+export KUBERNETES_VERSION=${KUBERNETES_VERSION:-"v1.22.0"}
+export UPGRADED_K8S_VERSION=${UPGRADED_K8S_VERSION:-"v1.22.0"}
```

Comment:
Please consider the changes in #766 if it might be conflicting

Reply:
@kashifest it doesn't have to be on the same PR, but I'm just pointing it out since you are touching these vars. We now have [1.22.1](https://github.com/kubernetes/kubernetes/releases/tag/v1.22.1) release of K8S. So, once we switch to 1.22 for integration tests, we can also switch upgrade tests too to run from 1.22.0 -> 1.22.1.